# User Story 1 â€“ Identifying virus resistant PGRs using Pretzel


Overview of Key Steps To Visualise Results in Pretzel

1. Project genetic map QTL into Cameor v1 genome assembly

2. Integrate genes and KASP markers

3. Identify sbm-1 gene with BLAST search

4. Examine WGS genotypes around the sbm-1 gene to identify haplotype shared by resistant accessions

5. Identify Multispecies Pulse 30K SNP array marker(s) that tag the haplotype

6. Examine PGRs from the AGG using the identified marker(s) to identify PGRs likely carrying the PSbMV resistance allele


## Login

Log in with the provided details:

Email Address (username)

    UserStory1@AGG
Password

    UserStory1

!!! note

    Please use the provided login account so you have access to all the relevant data 

![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-0.png)

## Video tutorial

<iframe width="1020" height="630" src="https://www.youtube.com/embed/7bd6uX_NvZw?si=lrJGFGMdBA3akTV9" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>

## Step by step with screenshots

### Load genetic map and QTL
Navigate to the explore tab and scroll down to the Datasets box find

    Field pea PSbMV QTL mapped in Kaspa x Yarrum

and select

    Ps VI
![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-1.png)
### Load SNP marker locations in the genome assembly
In the same explore tab and Datasets box find

    Field pea genetic map SNP markers anchored to genome assembly (Cameor v1)
and select 

    chr1LG6
![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-2.png)
### Invert the orientation of the axis
In the centre view click on the title of the axies at the top

    Kaspa x Yarrum : Ps IV
This will bring up the axis title menu on screen. Click the middle button to invert the orientation of the axis
![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-3.png)
To close this box press the top right "x" button 
![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-4.png)

### (Optional) Display the position and name of QTL
On the left axis locate the 3 boxes and click on one of them
![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-5.png)
An arrow should appear on left of the axies and click the arrow to bring the up name
![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-6.png)

### (Optional) Resize the display
Navigate to the View tab, scroll down till you find the slider titled and reduce this value to increase the distance between axes in the visualisation

    Outside Axis Margin
![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-7.png)

### Displaying features within the axis
In the centre view click on the title of the right axis at the top to bring up the same menu from earlier. This time, click on the right most button to display features. Then close it using the same "x" button in the top right hand corner.

![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-8.png)

### Load KASP marker positions in the genome assembly

![enter image description here](https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/user-stories/user-story-1/us-1-9.png)

### Locate sbm-1 gene in genome assembly using BLAST search

    >AY423375.2
    GGAGAAAGAAACCGAGAGAGAGCAAAAATGGTTGTAGAAGAAACCCCCAAATCCATCATCACCGACGATCAAATCACAACAAACCCTAATCGCGTTATCGAAGACGACAACAATCTTGAAGAAGGAGAGATCCTCGATGAAGACGATTCCTCCGCCACTTCCAAACCCGTCGTCCACCAACCTCACCTCCTCGAGAATTCTTGGACTTTCTGGTTTGATACCCCCGCAGCAAAATCCAAACAAGCCGCTTGGGGTAGCTCAATGCGACCCATCTACACTTTCTCCACTGTTGAAGAGTTTTGGAGCATTTACAATAACATTCATCATCCTGGTAAGTTGGCTGTGGGAGCAGATTTCTATTGTTTCAAGCATAAAATTGAACCTAAATGGGAGGATCCCATTTGTGCTAATGGTGGGAAATGGACTGCGAACTATCCGAAGGGAAAATCTGATACCAGTTGGTTATACACGTTGTTGGCAATGATTGGAGAACAATTTGATCATGGAGATGAAATTTGCGGAGCGGTTGTGAATGTAAGGGGTAGGGCTGAGAAGATTTCTATTTGGACTAAGAATGCTTCAAATGAAGCTGCTCAGGTGAGCATTGGAAAACAGTGGAAGGAGTTTCTTGATTATAATGAGACCATGGGCTTTATATTTCATGATGATGCAAGGAAACTCGACAGAAATGCTAAAAACAAATATGTTGTGTGAACTGTATTGCGTTCTTACATGGTAGCAAACTAGCAATTGCATGAGATGCCTCTCCGATATTCAACATGTTGCTTAATGCTTTCTAAGCCTTTTAAATCTCGTATTGAGTAGTATTTCCAGATTTGTGTGCGGATAATCTTTTGACTGTAGACGATGTTTCATCAATAATAGAGTGATTTAGTCAAAAAAAAAA

### (optional) Inspect BLAST results

### Zoom to region around sbm-1 gene

### Load genome assembly gene annotation

### Load whole genome sequence (WGS) based SNP data from resistant and susceptible accessions

### Select (brush) region around gene

### Switch to Genotypes tab in right panel

### Load genotype data for resistant and susceptible accessions

### Sort accessions based on haplotype around the KASP marker position

### Load Multispecies Pulse 30K SNP array AGG field pea PGR genotype dataset

### Identify SNP on Pulse 30K array that tags identified haplotype

### Remove WGS data from the view

### Load genotypes from AGG field pea PGR

### Sort accessions based on allele at the identified SNP